Sources/SpeziHealthKit/CollectSample/CollectSample.swift

```diff
@@ -10,11 +10,14 @@ import HealthKit
 import Spezi
 
 
-/// Collects a specified `HKSampleType`  in the ``HealthKit`` module.
+/// Collects a specified ``HealthKitSampleType``  via the ``HealthKit`` module.
 ///
-/// This structure define what and how the ``HealthKit`` samples are collected. By default, all samples of the provided `HKSampleType` will be collected. The collection starts on calling ``HealthKit/triggerDataSourceCollection()`` if you configure the `deliverySetting` as    ``HealthKitDeliverySetting/manual(safeAnchor:)`` or automatic once the application is launched when you configure anything else than manual, i.e.  ``HealthKitDeliverySetting/anchorQuery(_:saveAnchor:)`` or ``HealthKitDeliverySetting/background(_:saveAnchor:)``.
+/// This structure define what and how the ``HealthKit`` samples are collected. By default, all samples of the provided ``HealthKitSampleType`` will be collected.
+/// The collection starts on calling ``HealthKit/triggerDataSourceCollection()`` if you configure the `deliverySetting` as    ``HealthKitDeliverySetting/manual(saveAnchor:)`` or automatic once the application is launched when you configure anything else than manual, i.e.  ``HealthKitDeliverySetting/anchorQuery(_:saveAnchor:)`` or ``HealthKitDeliverySetting/background(_:saveAnchor:)``.
```

Comment:
We might want to clarify here that (automatic) collection only starts once the permissions have been requested authorization. Maybe we should add a DocC hint/tip box here to point to the function to request authorization from HealthKit?